epub33/reports/epub-properties-use.md

```diff
@@ -0,0 +1,604 @@
+# EPUB 3.3 Metadata Usage Report
+
+## Candidate Recommendation Exit Criteria
+
+The EPUB Working Group intends to exit the Candidate Recommendation stage and submit
+the EPUB 3.3 specification for consideration as a W3C Proposed Recommendation
+after documenting implementation of each feature.
+
+For this specification to advance to Proposed Recommendation, it has to be
+proven that metadata defined and required in this specification have sufficient usage by the
+target communities. This metadata falls into two categories:
+
+1. metadata for expressing information about the publication in the package document; and
+2. metadata for expressing preferred rendering of the content
+
+Usage of these properties means that they are regularly included in the Package Document
+metadata for their EPUB Publications (as appropriate for each title).
+
+## Publisher Implementations
+
+### Meta Properties Vocabulary
+
+The following table lists publishers who have stated that they are currently using
+the meta properties in production.
+
+<table>
+    <thead>
+        <tr>
+            <th>Property</th>
+            <th>Used By</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <th>alternate-script</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>authority</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>belongs-to-collection</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>collection-type</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>display-seq</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>file-as</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>group-position</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>identifier-type</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>meta-auth (deprecated)</th>
+            <td>
```

Comment:
I would think we could leave this one out. There is no specification for it in our document; it is redirected to EPUB 3.0 after all...